test-in-k8s.sh

```diff
@@ -9,21 +9,16 @@ kubectl get ns kubernetes-plugin-test || kubectl create ns kubernetes-plugin-tes
 kubectl get ns kubernetes-plugin-test-overridden-namespace || kubectl create ns kubernetes-plugin-test-overridden-namespace
 kubectl config set-context --current --namespace=kubernetes-plugin-test
 port_offset=$RANDOM
-http_port=$((2000 + $port_offset))
-tcp_port=$((2001 + $port_offset))
+http_port=$((2000 + port_offset))
+tcp_port=$((2001 + port_offset))
 kubectl delete --ignore-not-found --now pod jenkins
 sed "s/@HTTP_PORT@/$http_port/g; s/@TCP_PORT@/$tcp_port/g" < test-in-k8s.yaml | kubectl apply -f -
 kubectl wait --for=condition=Ready --timeout=15m pod/jenkins
+# Copy temporary split files
+tar cf - "$WORKSPACE_TMP" | kubectl exec -i jenkins -- tar xf -
```

Comment:
Confused about how this works, without a `-C` arg or anything. Are we sharing absolute path names between the agent VM and the pod?

Reply:
we're root inside `jenkins` container so we're effectively keeping the same absolute path. This avoids an extra path translation step when passing include/exclude options.

Reply:
And `tar x` just creates the missing directories like `/home/jenkins/workspace/es-plugin-PR_123@tmp`? Weird but OK if it works.

Reply:
tar cf creates a tar with full path references, so we unpack these under `/` in the containe.r